storage/reads/array_cursor_gen_test.go

```diff
@@ -146,7 +146,7 @@ func TestNewWindowAggregateArrayCursorMonths_Float(t *testing.T) {
 			Type: datatypes.AggregateTypeCount,
 		}
 
-		got, _ := newWindowAggregateArrayCursor(context.Background(), agg, window, &MockFloatArrayCursor{})
+		got, _ := NewWindowAggregateArrayCursor(context.Background(), agg, window, &MockFloatArrayCursor{})
```

Comment:
This file is generated, the source `tmpl` needs updated too

Reply:
Done